okhttp-tests/src/test/java/okhttp3/CallTest.java

```diff
@@ -1400,6 +1400,41 @@ private void postBodyRetransmittedAfterAuthorizationFail(String body) throws Exc
     assertEquals("GET /page2 HTTP/1.1", page2.getRequestLine());
   }
 
+  @Test public void getClientRequestTimeout() throws Exception {
+    enqueueRequestTimeoutResponses();
+
+    Response response = client.newCall(new Request.Builder()
+        .url(server.url("/")).build()).execute();
+
+    assertEquals("Body", response.body().string());
+  }
+
+  private void enqueueRequestTimeoutResponses() {
```

Comment:
Mostly off topic . . . though it causes duplication, I’m generally a fan of just inlining methods like these. For production code DRY is good. But for tests, I prefer to keep tests self-contained over keeping ’em not-repetitive.

That said, it's matters of degree. Lots of our test suite is built with supporting classes - even MockWebServer started out as a humble test utility here.